### DISCUSSION TITLE
Curso FullStack .NET - Fundamentos dos Minimal API's > Urls e Endpoints

### DISCUSSION THREAD

**Discussion by VictorTine** [balta-io/forum#626] [Tira Dúvidas] [answered]
Balta,

Tive problemas para rodar a aplicação, na secção de "Documentação da API", decidi então voltar algumas aulas e notei um erro relacionado ao "WebApplication". Mesmo na versão mais "simples" do código.
![Captura de tela 2024-07-17 223205](https://github.com/user-attachments/assets/12e6f69d-66c5-45a6-8a6e-474d04aa76bd)

**Comment by FBODeveloper** [comment, marked as answer]
Não seria apenas
var app = builder.Build();
